### ISSUE TITLE
Diagnostic LC0076 not raised with namespaces

### ISSUE THREAD

**Issue by Arthurvdv** [StefanMaron/BusinessCentral.LinterCop#917]
Seems that's there something wrong with the support for namespaces.

Both scenario's below don't raise the LC0076 diagnostic.

```AL
table 50100 MyTable
{
    fields
    {
        field(1; MyField; Integer) { }

        field(2; "VAT Product Posting Group 1"; Code[10])
        {
            TableRelation = Microsoft.Finance.VAT.Setup."VAT Product Posting Group".Code;
        }
        field(3; "VAT Product Posting Group 2"; Code[10])
        {
            TableRelation = Microsoft.Finance.VAT.Setup."VAT Product Posting Group";
        }
    }
}
